### PR TITLE
i fixed your website

### PR DESCRIPTION
**Pull request by exsersewo**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,6 +2,9 @@
 <meta charset="UTF-8">
 
 <html>
+	<head>
+		<meta http-equiv="refresh" content="2; URL='http://google.com/'">
```
Comment (Multivax) [inline index.html]: no 🛑

Comment (exsersewo) [reply index.html]: image

Comment (Multivax) [reply index.html]: 👎

Comment (exsersewo) [reply index.html]: image

**Comment by Multivax** [comment]
not allowed

**Comment by Aveoxya** [comment]
@Multivax @exsersewo i thought this was serious repo 🤔 🤔 🤔 🤔 
tenor

**Comment by Multivax** [comment]
github how to delete pull request

**Comment by Multivax** [comment]
oh shit this isnt bing.go

**Comment by Aveoxya** [comment]
ghoegel help this man plez

**Pull request closed without merging**